src/Configuration/ValueObject/SSLConnectionConfig.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\DbExtractorConfig\Configuration\ValueObject;
+
+class SSLConnectionConfig implements ValueObject
+{
+    private ?string $key;
+
+    private ?string $cert;
+
+    private ?string $ca;
+
+    private ?string $cipher;
+
+    private bool $verifyServerCert;
+
+    public static function fromArray(array $data): self
+    {
+        $ssl = $data['ssl'];
+        return new self(
+            $ssl['key'] ?? null,
+            $ssl['cert'] ?? null,
+            $ssl['ca'] ?? null,
+            $ssl['cipher'] ?? null,
+            $ssl['verifyServerCert'] ?? true
+        );
+    }
+
+    public function __construct(?string $key, ?string $cert, ?string $ca, ?string $cipher, bool $verifyServerCert)
+    {
+        $this->key = $key;
+        $this->cert = $cert;
+        $this->ca = $ca;
+        $this->cipher = $cipher;
+        $this->verifyServerCert = $verifyServerCert;
+    }
+
+    public function getKey(): ?string
```

Comment:
Tomas minule pisal, ze by sme nemali pouzivat na takomto mieste `?string` ... ale iba `string` a pridat `has*` metodu ... inak s tym bude kopec problemov v PhpStan a je to menej ciste ... stretol som sa s tymto napadom vtedy prvy krat .... ale dava zmysel a kod je potom ovela menej nachylny na nejaku neosetrenu null hodnotu ... v tomto repe je tak spraveny `ExportConfig`, atd...

Reply:
Jj přidal jsem to tam